CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## [0.6.2] 2018-01-05
```

Comment:
If we are changing the version, you will also need to update the DESCRIPTION file as well.

Reply:
The version is already 0.6.2 in DESCRIPTION file